Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -75,6 +75,18 @@ private static string RoutingRunTimeVersion
             get { return System.Environment.GetEnvironmentVariable(Constants.RoutingRunTimeVersion); }
         }
 
+        // LOGICAPP_URL = [url to PUT logicapp.json to]
+        private static string LogicAppUrl
+        {
+            get { return System.Environment.GetEnvironmentVariable(Constants.LogicAppUrlKey); }
+        }
+
+        // %HOME%\site\wwwroot\logicapp.json
+        private static string LogicAppJsonFilePath
+        {
+            get { return System.Environment.ExpandEnvironmentVariables(@"%HOME%\site\wwwroot\" + Constants.LogicAppJson); }
```

Comment:
Yes we could.   However, this helper is the common routine (called by Kudu and msdeploy) so we keep it simple avoiding any more dedendency for msdeploy.